.travis.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     # Build and test go
     - <<: *test
       name: Go on OpenShift
-      script: make test/ci-go
+      script: make test/ci-go ARGS="-v"
```

Comment:
I think it makes sense for CI and the amount of output doesn't really matter when running in CI. At least for this PR it makes sense because I want to be able to make sure that the Gopkg.toml is unmodified when running within our repo, which we can only see after we merge this PR.